### PR TITLE
Blood glucose

### PR DESCRIPTION
**Pull request by Gimpneek**
- Moving blood sugar into own module and renaming blood glucose
- Making blood glucose use the new means of drawing graphs

### CODE REVIEW

File: nh_blood_glucose/models/nh_clinical_patient_observation_blood_glucose.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    _description = "Blood Glucose"
+    _columns = {
+        'blood_glucose': fields.float('Blood Glucose', digits=(2, 1)),
+    }
+    _form_description = [
+        {
+            'name': 'blood_glucose',
+            'type': 'float',
+            'label': 'Blood Glucose (mmol/L)',
+            'min': 1.0,
```
Comment (moderatemisbehaviour) [inline nh_blood_glucose/models/nh_clinical_patient_observation_blood_glucose.py]: should be 0.0

File: nh_blood_glucose/tests/nh_clinical_patient_observation_blood_glucose/test_field_validation.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            })
+
+    def test_blood_glucose_above_max_raises_exception(self):
+        with self.assertRaises(ValidationError):
+            self.blood_glucose_model.create({
+                'patient_id': self.patient.id,
+                'blood_glucose': self.blood_glucose_max + 0.1
+            })
+
+    def test_blood_glucose_on_min_does_not_raise_exception(self):
```
Comment (moderatemisbehaviour) [inline nh_blood_glucose/tests/nh_clinical_patient_observation_blood_glucose/test_field_validation.py]: This test should have failed!

**Comment by moderatemisbehaviour** [comment]
Absolutely brilliant :1st_place_medal: :shipit:

**Comment by Nayira** [comment]
Tests passed for Stories 898, 97 & 918, Feature branch bugs 1149, 1150, 1151 & 1152, and UI/UX stories 979, 950 & 936. :shipit:

**Pull request merged**